deploy/example_clusters/cr-cluster-cdh5142cm-gke.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: "kubedirector.bluedata.io/v1alpha1"
+kind: "KubeDirectorCluster"
+metadata:
+  name: "cdh5142cm-instance"
+spec:
+  app: cdh5142cm
+  serviceType: LoadBalancer
+  roles:
+  - id: controller
+    resources:
+      requests:
+        memory: "16Gi"
+        cpu: "2"
+      limits:
+        memory: "16Gi"
+        cpu: "2"
```

Comment:
Should we define some persistent storage?

Reply:
sure why not